Dockerfile.dev

```diff
@@ -1,7 +1,10 @@
-FROM python:3.7
+FROM python:3.8
 
-RUN apt-get update \
-    && apt-get install -y --no-install-recommends \
+RUN \
+    && apt-get update && apt-get install -y --no-install-recommends \
+		software-properties-common \
+    && add-apt-repository ppa:jonathonf/ffmpeg-4 \
```

Comment:
When building the container I'm getting this back
E: The repository 'http://ppa.launchpad.net/jonathonf/ffmpeg-4/ubuntu groovy Release' does not have a Release file.